api/users/serializers.py

```diff
@@ -102,6 +102,11 @@ def update(self, instance, validated_data):
                 setattr(instance, attr, value)
         try:
             instance.save()
+        # When a ValidationError is raised for 'social', e.args[0] will a
+        # dict of the form {'social': [<Exception>]}
+        except ValidationError as e:
+            if 'social' in e.args[0]:
+                raise InvalidModelValueError(detail=e.args[0]['social'][0][0])
```

Comment:
There's a chance we might be swallow a `ValidationError` here. It probably makes sense to add an `else:` block here that just raises the ValidationError that just raises.

Also, since `ValidationValueError` is a subclass of `ValidationError`, there is no way to get to the below `except ValidationValueError` block. Would it make sense to just catch `ValidationError`?

```python
try:
    ...
except ValidationError as e:
    if 'social' ...:
        raise ...
   else:
       raise ...
```